### ISSUE TITLE
Primgroup Centroids qL SOP overhaul

### ISSUE THREAD

**Issue by johnnyquest** [qLab/qLib#467]
- move from the very-old-skool group/pointindex-based stuff to name-based
- make sure it works with packeds
- this should became a version 2

UI overhaul: Add Attributes /
- tab order: From Primitives / Transform (should be renamed Transform Attribs) / Measure Pieces / Group Indexing
- "Group Indexing" tab should be last, Group Name Attribute should be swtiched off
- "From Primitives" should be enabled, and the attribs from prims linked to "Attrib Name" (name attribute)
- mode should be "Using Name Attribute" by default
- Transform / Orientation Vectors should have a Two-vectors and Orient (quaternion) mode; all attributes should be able to be specified by user (as well as rest position; make sure rest position is non-transformable float3)

**Comment by johnnyquest** [comment]
apparently this has been retired, lol, i'm closing this
